src/EditorFeatures/Core.Cocoa/Preview/PreviewPane.cs

```diff
@@ -95,7 +96,7 @@ public PreviewPane(string? id, string? title, Uri? helpLink, string? helpLinkToo
                 new Microsoft.VisualStudio.Text.SnapshotPoint(_differenceViewerPreview.Viewer.InlineView.TextSnapshot, 0));
         }
 
-        public PreviewPane(IntPtr ptr)
+        public PreviewPane(NativeHandle ptr)
```

Comment:
The IntPtr constructor worries me, especially since it's public. This class isn't resurrection-safe - as in there is no state that's saved in native so it can be restored later. Why is it needed?

Reply:
Not a thing to fix in this PR, just wanted to have a mental note that we should also check Roslyn when we do a reference sanity pass.